lib/rules/cli/4.1.yaml

```diff
@@ -1372,3 +1372,26 @@
   :options:
     :skip-check: --skip-check=<value>
     :z: -z default
+:create_cronjob:
```

Comment:
Please place the command in alphabetical order.

Reply:
Done